src/codeu/chat/util/encryption/EncryptorTest.java

```diff
@@ -0,0 +1,41 @@
+import java.math.BigInteger;
+
+public class EncryptorTest {
```

Comment:
Good to write tests. Yaay! Could we make them junit based? I think Ruiqi had written some junit based tests, you can check with him or just google for more info.